tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -69,17 +70,22 @@ export class ScalarCardDataTable {
   getMaxValueInRange(
     points: ScalarCardPoint[],
     startPointIndex: number,
-    endPointIndex: number
+    endPointIndex: number,
+    smoothed: boolean = false
   ): number {
-    let maxValue = points[startPointIndex].value;
+    let maxValue = this.getValue(points[startPointIndex], smoothed);
     for (let i = startPointIndex; i <= endPointIndex; i++) {
-      if (maxValue < points[i].value) {
-        maxValue = points[i].value;
+      if (maxValue < this.getValue(points[i], smoothed)) {
+        maxValue = this.getValue(points[i], smoothed);
       }
     }
     return maxValue;
   }
 
+  getValue(point: ScalarCardPoint, smoothed: boolean) {
```

Comment:
nit: getValue is not very expressive of what value we are getting. Can we rename to something like "maybeSmoothValue"?